nsv13/code/controllers/subsystem/starsystem.dm

```diff
@@ -0,0 +1,199 @@
+//Subsystem to control overmap events and the greater gameworld
+SUBSYSTEM_DEF(starsystem)
+	name = "Starsystem"
+	wait = 10
+	flags = SS_NO_INIT
+	var/last_combat_enter = 0 //Last time an AI controlled ship attacked the players
+	var/modifier = 0 //Time step modifier for overmap combat
+	var/list/systems = list()
+	var/datum/starsystem/hyperspace //The transit level for ships
+	var/bounty_pool = 0 //Bounties pool to be delivered for destroying syndicate ships
+
+/datum/controller/subsystem/starsystem/fire() //Overmap combat events control system, adds weight to combat events over time spent out of combat
+	if(last_combat_enter + (5000 + (1000 * modifier)) < world.time) //Checking the last time we started combat with the current time

```

Comment:
A timer for this may be more effective

Reply:
Consider using MINUTES macros here for readability. I can't really process these numbers in my head.

Reply:
KMC and math

Reply:
Would you prefer it to be based on divisible by 600?